src/app/models/schema/user.schema.ts

```diff
@@ -25,4 +25,10 @@ export const createUserSchema: z.ZodSchema<CreateUserDto> =
         message:
           "Password must be at least 6 characters long",
       }),
+
+    createdAt: z.date(),
+
+    updatedAt: z.date(),
+
+    deletedAt: z.date(),
```

Comment:
[Q] This `createUserSchema` schema is for  validating which is the request body/payload `CreateUserDTO`.
 
The frontend wouldn't need to pass these fields. Why add it to the schema?